### DISCUSSION TITLE
How to add custom oauth providers?

### DISCUSSION THREAD

**Discussion by u07** [umputun/remark42#1562] [Ideas]
Hi! I'm curious, is it possible to manually add more oauth providers, popular among my users? It's nice to have yandex built-in, but I need also vk.com, ok.ru and mail.ru. Did not found that in docs, is there a mechanism to do that with remark42? If not, please consider as a wishlist proposition.

**Comment by paskal** [comment]
The support of these providers should be added to https://github.com/go-pkgz/auth. For that, I propose to start an issue there to get information if it would be ever done.
